src/hb-ot-stat-table.hh

```diff
@@ -230,18 +252,58 @@ struct STAT
   bool sanitize (hb_sanitize_context_t *c) const
   {
     TRACE_SANITIZE (this);
-    return_trace (likely (c->check_struct (this) &&
-			  majorVersion == 1 &&
-			  minorVersion > 0 &&
+    return_trace (likely (version.sanitize (c) &&
+                          c->check_struct (this) &&
+			  version.major == 1 &&
 			  designAxesOffset.sanitize (c, this, designAxisCount) &&
 			  offsetToAxisValueOffsets.sanitize (c, this, axisValueCount, &(this+offsetToAxisValueOffsets))));
   }
+  
+  bool has_data () const { return version.to_int (); }
+
+  unsigned get_design_axis_count () const { return designAxisCount; }
+
+  unsigned get_axis_record_name_id (unsigned int axis_record_index) const
```

Comment:
`hb_ot_name_id_t`.  Also, be consistent.  Just `unsigned` instead of `unsigned int` now that we're doing that.